features/step_definitions/journey_test_steps.rb

```diff
@@ -2074,8 +2084,51 @@ def check_latest_row_correct(service_status)
   else
     page.should have_no_selector(:xpath, "//a[contains(@href, '#{store.current_brief_id}')]/../../../td[3]/span[text()]")
   end
-end  
+end
 
 Then /^The (?:Organisation name|Requirements title) '(.*)' is presented on the page$/ do |name|
   page.should have_content(name)
 end
+
+Then /^I am taken back to the "Overview of work" page with the (.*) question and answer listed under "Clarification questions"$/ do |question_number|
+  page.should have_selector(:xpath, ".//div[@class='marketplace-paragraph']/h2[contains(text(), 'Overview of work')]")
+  current_url.should end_with("#{dm_frontend_domain}/buyers/frameworks/#{store.framework}/requirements/#{store.lot}/#{store.current_brief_id}#clarification-questions")
+
+  case question_number
+  when 'first'
+    store.question_count = 1
+   page.should have_selector(:xpath, ".//div/h2[contains(text(), 'Clarification questions')]/..//tbody/tr[#{store.question_count}]/td[1]//span[contains(text(), '#{store.question_count}.')]/../../span[contains(text(),'#{@value_of_interest['question']}')]")
+   page.should have_selector(:xpath, ".//div/h2[contains(text(), 'Clarification questions')]/..//tbody/tr[#{store.question_count}]/td[2]//span[contains(text(),'#{@value_of_interest['answer']}')]")
+   store.question_1 = @value_of_interest['question']
+   store.answer_1 = @value_of_interest['answer']
+
+  when 'second'
+    store.question_count = 2
+    page.should have_selector(:xpath, ".//div/h2[contains(text(), 'Clarification questions')]/..//tbody/tr[#{store.question_count}]/td[1]//span[contains(text(), '#{store.question_count}.')]/../../span[contains(text(),'#{@value_of_interest['question']}')]")
+    page.should have_selector(:xpath, ".//div/h2[contains(text(), 'Clarification questions')]/..//tbody/tr[#{store.question_count}]/td[2]//span[contains(text(),'#{@value_of_interest['answer']}')]")
+    store.question_2 = @value_of_interest['question']
+    store.answer_2 = @value_of_interest['answer']
+
+  else
+    fail("The value provided is not valid: #{question_number}")
+  end
+
+  page.all(:xpath, "//div/h2[text()][contains(text(), 'Clarification questions')]/..//tbody/tr").length.should == store.question_count
+end
+
+Then /^I am taken to the list of "Supplier opportunities" page$/ do
+  page.should have_selector(:xpath, "//h1[contains(text(), 'Supplier opportunities')]")
+  page.should have_selector(:xpath, ".//*[@id='global-breadcrumb']/nav/*[@role='breadcrumbs']/li[1]//*[contains(text(), 'Digital Marketplace')]")
```

Comment:
I think this is a really good general purpose test that we've got to the right page, I think this is all that needs to be checked for many pages - maybe re-use this instead of making a specific `I am taken to the list of` test:
https://github.com/alphagov/digitalmarketplace-functional-tests/blob/master/features/step_definitions/ssp_steps.rb#L139